orcid-core/src/main/java/org/orcid/core/adapter/v3/converter/WorkContributorsConverter.java

```diff
@@ -77,9 +82,13 @@ public WorkContributors convertFrom(String source, Type<WorkContributors> destin
         final ObjectMapper objectMapper = new ObjectMapper();
         List<WorkContributorsList> langList = new ArrayList<>();
         try {
-            langList = objectMapper.readValue(source, new TypeReference<List<WorkContributorsList>>(){});
+        	//TODO: Dani, looks like this is not doing anything? It is just an empty list
```

Comment:
@amontenegro here is where the new list of top contributors is mapped. I have even created a new Object Called WorkContributorList since the query is retrieving now a JSON array